conf/default-colors.yaml

```diff
@@ -20,3 +20,6 @@ color:
   button-primary-bg: 'rgba(75, 157, 249, 1)'
   button-primary-color: '{{color.white}}'
   icons: '#333'
+  message-info: '#f0f0f0'
+  message-success: '#dff0d8'
+  message-error: '#f2dede'
```

Comment:
might want to change these names to be a bit more generic since they'll be used in text elements, alert boxes, notifications, form validation, etc.